second_test/second_test.spec.js

```diff
@@ -0,0 +1,37 @@
+// @ts-check
```

Comment:
This can also be removed since it's autogenerated referring to Typescript, however you are writing that in Javascript.